samples/KestrelSample/KestrelSample.csproj

```diff
@@ -8,12 +8,9 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <PackageReference Include="Microsoft.AspNetCore.Server.Kestrel.Transport.Libuv" Version="$(AspNetCoreVersion)" />
```

Comment:
Correct. The libuv transport is only a nuget package and not part of the shared framework IIRC.

Reply:
Yes, it it were part of the shared framework it would be a pain for source-build because it requires building a cross-platform package (https://www.nuget.org/packages/Libuv/).